### PR TITLE
[RM Ch07] Add ETSI NFV to the list of common standards

### LINKED ISSUE #431: [RM Ch07] Modifications needed for section 7.10.5
Following discussion we had during RM chapter 4 meeting the 16th of October,
the content of section 7.10.5 needs to be modified for a better integration in the Security Chapter.
The 2 tables were initially moved from chapter 4: text must be rephrased, especially to capture what is expected at a model level.

@ASawwaf I can create a PR for that

### PR DESCRIPTION
**Pull request by glimonta**
Fixes #431

### CODE REVIEW

**Comment by rabi-abdel** [comment]
@karinesevilla in issue #431 you highlighted there is a need to do some Changs to 7.10.5. can we use this PR by @glimonta to reflect that change too?

File: doc/ref_model/chapters/chapter07.md [hunk 1]
```diff
@@ -43,7 +43,9 @@
   * [7.10.1 Networking Security Zoning.](#7.4.1)
   * [7.10.2 Encryption.](#7.4.2)
   * [7.10.3 Platform Patching.](#7.4.3)
-  * [7.10.4 Boot Integrity Measurement (TPM).](#7.4.4)
+  * [7.10.4 Boot Integrity Measurement (TPM).](#7.10.4)
+  * [7.10.5 Runtime Integrity Measurement (TPM).](#7.10.5)
+  * [7.10.6 NFVI & VIM.](#7.10.6)
```
Comment (deebhatia) [inline doc/ref_model/chapters/chapter07.md]: Remove period from headings. Please delete it from the existing ones in this chapter. I will also request to fix the references of § 7.8-§ 7.10 in the headings.

File: doc/ref_model/chapters/chapter07.md [hunk 2]
```diff
@@ -647,10 +651,21 @@ NFVI operators should ensure that the platform including the components (hypervi
...
+
+Additionally, NFVI Operators should ensure that OS kernel measurements can be recorded by using a TPM-aware bootloader (e.g. [tboot](https://sourceforge.net/projects/tboot/) or [shim](https://github.com/rhboot/shim)), which can extend the root of trust up to the kernel level.
+The validation of the platform measurements can be performed by TPM’s launch control policy (LCP) or through the remote attestation server.
 
 <a name="7.10.5"></a>
-### 7.10.5 NFVI & VIM
+### 7.10.5 Runtime Integrity Measurement (TPM)
+If a remote attestation server is used to monitor platform integrity, the operators should ensure that attestation is performed periodically or in a timely manner.
+Additionally, platform measurements may be extended to monitor the integrity of the static filesystem at run-time by using a TPM aware kernel module, such as [Linux IMA (Integrity Measurement Architecture)](https://sourceforge.net/p/linux-ima/wiki/Home/) for linux platforms, or by using the [trust policies](https://github.com/opencit/opencit/wiki/Open-CIT-3.2-Product-Guide#88-trust-policies) funcitonality of OpenCIT.
```
Comment (deebhatia) [inline doc/ref_model/chapters/chapter07.md]: * funcitonality -> functionality

File: doc/ref_model/chapters/chapter07.md (hunk 1 above)
Comment (glimonta) [reply doc/ref_model/chapters/chapter07.md]: Fixed in the last commit.

File: doc/ref_model/chapters/chapter07.md (hunk 2 above)
Comment (glimonta) [reply doc/ref_model/chapters/chapter07.md]: Fixed in the last commit.

**Comment by rabiabdel** [comment]
@ASawwaf @karinesevilla @deebhatia please review

**Comment by CsatariGergely** [comment]
Fixes #597

**Pull request merged**